### ISSUE TITLE
the Hyper-parameters of the Classification Net

### ISSUE THREAD

**Issue by JaringAu** [ElementAI/wise_ils#3]
Hi,

we use the default parameters of PRM (https://github.com/ZhouYanzhao/PRM/blob/pytorch/demo/config.yml) to train the classification network, (change the train_splits from trainval to trianaug, of course). However, the quality of the results we got are slightly worse than that of the points in 'datasets/prm_point_dict.json'.
Therefore, could you share your hyper-parameter setting of classification net?

By the way, we have tried several times, but can't acheive the performance reported in PRM* (mAP50: 26.8). only get 11.5 mAP50 with MCG proposals and 21.5 mAP50 with COB proposals.
Have you ever repoduced their work successfully? could you give some advices to us?

Thanks a lot.

*Zhou, Yanzhao, et al. "Weakly supervised instance segmentation using class peak response." Proceedings of the IEEE Conference on Computer Vision and Pattern Recognition. 2018.

**Comment by IssamLaradji** [comment]
1. The file I used is https://github.com/ElementAI/wise_ils/blob/master/models/prm.py for training prm. I used the `PRM` class and the `PRMLoss` function to compute the loss.

2. I remember having the same difficulty reproducing those results. Perhaps you could ask them in their Github repo?

Best!

**Comment by IssamLaradji** [comment]
I will upload the pretrained PRM model soon.

**Comment by JaringAu** [comment]
Got it!
Thanks a lot for your comments:)

**Comment by IssamLaradji** [comment]
you welcome! :)
